troubleshooting/ETW/Readme.md

```diff
@@ -116,21 +116,24 @@ StatusMonitor uses TraceEventSession to record ETW logs.
 - https://github.com/dotnet/roslyn/wiki/Recording-performance-traces-with-PerfView
 - https://github.com/microsoft/perfview/blob/master/src/TraceEvent/TraceEventSession.cs
 
-### Self Diagnostics
+### Self-Diagnostics (available in version 2.18.0)
```

Comment:
The first sentence already noted "As of version 2.18.0", but I don't have strong opinion on having it redundant either.